web/_js/atlas.js

```diff
@@ -1298,6 +1298,9 @@ var atlas = [
 { "id": "32aae30a-r-grandorder-abby", "name": "Abby | Abigail Williams", "description": "A character from the mobile game Fate/Grand Order.\n \nA Foreigner-class Servant and a key character in the events of Salem", "website": "https://typemoon.fandom.com/wiki/Abigail_Williams", "subreddit": "/r/grandorder", "center": [ 946.5, 1426.5 ], "path": [ [ 941.5, 1419.5 ], [ 941.5, 1433.5 ], [ 951.5, 1433.5 ], [ 951.5, 1419.5 ], [ 941.5, 1419.5 ] ] },
 { "id": "ed0ac5cf-r-fatestaynight-fate", "name": "Saber | Fate Route", "description": "Saber - a Main character in the Visual Novel Fate/Stay Night\n\nFate Route - One of the main routes in the visual novel where Saber is featured as the main heroine.", "website": "https://typemoon.fandom.com/wiki/Saber", "subreddit": "/r/fatestaynight/", "center": [ 18.5, 1046.5 ], "path": [ [ 0.5, 1028.5 ], [ 35.5, 1028.5 ], [ 35.5, 1063.5 ], [ 0.5, 1063.5 ], [ 0.5, 1028.5 ] ] },
 { "id": "c0d29bf9-r-grandorder-emiya", "name": "EMIYA", "description": "A main character in the Fate/Stay Night series and also a summonable character in Fate/Grand Order", "website": "https://typemoon.fandom.com/wiki/EMIYA_(Archer)", "subreddit": "/r/grandorder", "center": [ 8.5, 957.5 ], "path": [ [ 0.5, 942.5 ], [ 16.5, 942.5 ], [ 15.5, 972.5 ], [ 0.5, 972.5 ], [ 0.5, 942.5 ] ] },
+{ "id": 472, "submitted_by": "Xephorix512", "name": "Yoshi", "description": "Yoshi is a popular character from the Super Mario Bros. franchise", "website": "", "subreddit": "r/Mario", "center": [1846.5, 1267.5], "path": [[1835.5, 1258.5], [1856.5, 1258.5], [1856.5, 1276.5], [1836.5, 1276.5]] },
+{ "id": 473, "submitted_by": "Xephorix512", "name": "Halo Games", "description": "Art depicting several main characters from the Halo universe.", "website": "https://www.xbox.com/en-US/games/halo", "subreddit": "r/halo", "center": [359.5, 1197.5], "path": [[343.5, 1172.5], [374.5, 1172.5], [374.5, 1221.5], [343.5, 1221.5]] },
+{ "id": 474, "submitted_by": "Xephorix512", "name": "Crystal Heart", "description": "A crystal heart from the game "Celeste"", "website": "", "subreddit": "r/celestegame", "center": [1660.5, 557.5], "path": [[1659.5, 553.5], [1663.5, 551.5], [1666.5, 550.5], [1668.5, 552.5], [1668.5, 559.5], [1659.5, 566.5], [1652.5, 559.5], [1652.5, 552.5], [1655.5, 550.5], [1658.5, 552.5]] }
```

Comment:
```suggestion
{ "id": 474, "submitted_by": "Xephorix512", "name": "Crystal Heart", "description": "A crystal heart from the game \"Celeste\"", "website": "", "subreddit": "r/celestegame", "center": [1660.5, 557.5], "path": [[1659.5, 553.5], [1663.5, 551.5], [1666.5, 550.5], [1668.5, 552.5], [1668.5, 559.5], [1659.5, 566.5], [1652.5, 559.5], [1652.5, 552.5], [1655.5, 550.5], [1658.5, 552.5]] }
```

Currently the website doesn't work, you need to escape the `"`